src/actions/vstudio/_vstudio.lua

```diff
@@ -338,7 +338,7 @@
 		end
 
 		-- Then the system libraries, which come undecorated
-		local system = config.getlinks(cfg, "system", "fullpath")
+		local system = config.getlinks(cfg, "system", "name")
```

Comment:
actually looking at the change a little bit more, if you provide this:
```lua links { "../../bin/somelib" }```

then this change will make the code act as if you did:

```lua
libdirs { "../../bin" }
links { "somelib" }
```

so it doesn't fail, it just works differently, but I guess that could lead to issues if you had duplicate names or something...